ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -1964,4 +1964,37 @@ def test_tensorflow_sinh(
         fn_tree=fn_tree,
         on_device=on_device,
         x=x[0],
-    )
\ No newline at end of file
+    )
+
+
+# zeta
+@handle_frontend_test(
+    fn_tree="tensorflow.math.zeta",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=[
+            "float32",
+            "float64",
+        ],
```

Comment:
As long as we have defined the supported dtypes for the functions using the appropriate decorators, we don't need to define a limited set of available_dtypes in the test. We should be testing for all valid dtypes.